src/fluvio.test.ts

```diff
@@ -168,7 +165,7 @@ describe('Configures a SmartModule with a Filter configuration', () => {
             JSON.parse(serverLogsFile)
         const stream = await consumer.streamWithConfig(Offset.FromBeginning(), {
             smartmoduleType: SmartModuleType.Filter,
-            smartmoduleData: wasmSmartModule.toString('base64'),
```

Comment:
Could you create a test for the `smartmoduleData` case?

Reply:
Of course!

Reply:
Okey both test cases are now into the same test suite so we can reuse calls to `beforeEach` and `afterEach` which will reset state to initial on every test case. Check it out here: [9371610](https://github.com/infinyon/fluvio-client-node/pull/263/commits/937161098d214c5192fc2b06b0c1fb82b3e9370c)